lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -775,11 +775,11 @@
     </datatype>
     <datatype extension="pdbqt" type="galaxy.datatypes.molecules:PDBQT" display_in_upload="true"/>
     <datatype extension="pqr" type="galaxy.datatypes.molecules:PQR" display_in_upload="true" />
-    <datatype extension="cell" type="galaxy.datatypes.molecules:Cell" display_in_upload="true"/>
-    <datatype extension="cif" type="galaxy.datatypes.molecules:CIF" display_in_upload="true"/>
+    <datatype extension="cell" type="galaxy.datatypes.molecules:Cell" display_in_upload="true" description="CASTEP input file contains geometry information, including cell parameters, atomic coordinates, and k-point coordinates and weights. The file consists of lines containing keywords and data blocks, and associated values"/>
```

Comment:
```suggestion
    <datatype extension="cell" type="galaxy.datatypes.molecules:Cell" display_in_upload="true" description="The .cell file contains geometry information, including cell parameters, atomic coordinates, and k-point coordinates and weights. The file consists of lines containing keywords and data blocks, and associated values" description_url="https://www.tcm.phy.cam.ac.uk/castep/documentation/WebHelp/content/modules/castep/expcastepfilecell.htm"/>
```

Don't know if you forgot to add the url since most of the description seems to come from there. I'm also slightly paranoid about calling the `.cell` an "input file" since that's how CASTEP describes the `.param` format.

Reply:
Noted will include the link and push changes